bign256/src/arithmetic/scalar.rs

```diff
@@ -154,7 +154,7 @@ impl Invert for Scalar {
 
 impl IsHigh for Scalar {
     fn is_high(&self) -> Choice {
-        const MODULUS_SHR1: U256 = BignP256::ORDER.shr_vartime(1);
+        const MODULUS_SHR1: U256 = BignP256::ORDER.wrapping_shr_vartime(1);
```

Comment:
I tried using `.shr(1)` and `.shr(1u32)` here (and for the same constant in other curves) but I just can't get it to use the `const fn` over the trait:

```
error[E0015]: cannot call non-const fn `<elliptic_curve::crypto_bigint::Uint<4> as Shr<u32>>::shr` in constants
   --> bign256/src/arithmetic/scalar.rs:157:52
    |
157 |         const MODULUS_SHR1: U256 = BignP256::ORDER.shr(1u32);
    |                                                    ^^^^^^^^^
    |
    = note: calls in constants are limited to constant functions, tuple structs and tuple variants
```

Reply:
Curiously, it works just fine here: https://github.com/RustCrypto/crypto-bigint/blob/5ff769f/src/lib.rs#L56

Reply:
Well, `shl_vartime` works, now that I've added it (back)